website/docs/developer/research/research-graphql.md

```diff
@@ -237,6 +286,18 @@ The following tools might be useful for developers while working with GraphQL
 - GraphQL Boilerplates- useful for backend projects.
 - Apollo Graph Manager-  useful for monitoring the performance and usage of GraphQL backend.
 
+### Recommendation
+While GraphQL is an interesting technology, the lack of documentation makes implementation difficult.
+In particular, deployment and integration lack vital instructions, with most tutorials running in a docker container or Python script rather than a realistic environment.
+Theoretically GraphQL could be run in an EC2 instance with some form of connecting layer such as [Fargate](https://aws.amazon.com/fargate/?whats-new-cards.sort-by=item.additionalFields.postDateTime&whats-new-cards.sort-order=desc&fargate-blogs.sort-by=item.additionalFields.createdDate&fargate-blogs.sort-order=desc) but this is untested and would need research and cost analysis.
```

Comment:
By connecting layer do you mean orchestration service? (i.e. ECS, Fargate, EKS)

Reply:
I am unable to find references for the term 'Orchestration Service'.

Reply:
Ah. No, that is not what I meant. Updated.